diff --git a/lib/configs/flat/recommended.js b/lib/configs/flat/recommended.js
index cc0668c77de2e65f0ba9feb67d0c95f9d807a5a3..5eb6b02cf3133de6a22f1eb89a6a341f91fd933b 100644
--- a/lib/configs/flat/recommended.js
+++ b/lib/configs/flat/recommended.js
@@ -16,7 +16,6 @@ const config = [
             'svelte/no-dupe-use-directives': 'error',
             'svelte/no-export-load-in-svelte-module-in-kit-pages': 'error',
             'svelte/no-immutable-reactive-statements': 'error',
-            'svelte/no-inner-declarations': 'error',
             'svelte/no-inspect': 'warn',
             'svelte/no-navigation-without-resolve': 'error',
             'svelte/no-not-function-handler': 'error',
diff --git a/lib/rules/no-inner-declarations.d.ts b/lib/rules/no-inner-declarations.d.ts
deleted file mode 100644
index 92ea50db003392a701e78fd026a0f4dfb65baf98..0000000000000000000000000000000000000000
diff --git a/lib/rules/no-inner-declarations.js b/lib/rules/no-inner-declarations.js
deleted file mode 100644
index c6023caf31f8355ee35023f3aad28907600f378f..0000000000000000000000000000000000000000
diff --git a/lib/rules/prefer-const.d.ts b/lib/rules/prefer-const.d.ts
deleted file mode 100644
index 92ea50db003392a701e78fd026a0f4dfb65baf98..0000000000000000000000000000000000000000
diff --git a/lib/rules/prefer-const.js b/lib/rules/prefer-const.js
deleted file mode 100644
index fc69aaf2c6c9f3599a8a039a10a6836703468c27..0000000000000000000000000000000000000000
diff --git a/lib/utils/eslint-core.js b/lib/utils/eslint-core.js
index 115228b92992ebf528dd66d4d4b2f83b28a272e2..bbd233bf47a113c33474241cab19d0f65ed4a9e8 100644
--- a/lib/utils/eslint-core.js
+++ b/lib/utils/eslint-core.js
@@ -60,7 +60,5 @@ export function getCoreRule(ruleName) {
     catch {
         // getRules() is no longer available in flat config.
     }
-    const { builtinRules } = require('eslint/use-at-your-own-risk');
-    ruleMap = builtinRules;
-    return builtinRules.get(ruleName) || null;
+    return null;
 }
diff --git a/lib/utils/rules.js b/lib/utils/rules.js
index 23f8682ee2b4eee2c6cb74f4a37db82cbfcd0c5e..028872566a2f2fc87ab3d32c38204d238ed36bda 100644
--- a/lib/utils/rules.js
+++ b/lib/utils/rules.js
@@ -30,7 +30,6 @@ import noGotoWithoutBase from '../rules/no-goto-without-base.js';
 import noIgnoredUnsubscribe from '../rules/no-ignored-unsubscribe.js';
 import noImmutableReactiveStatements from '../rules/no-immutable-reactive-statements.js';
 import noInlineStyles from '../rules/no-inline-styles.js';
-import noInnerDeclarations from '../rules/no-inner-declarations.js';
 import noInspect from '../rules/no-inspect.js';
 import noNavigationWithoutBase from '../rules/no-navigation-without-base.js';
 import noNavigationWithoutResolve from '../rules/no-navigation-without-resolve.js';
@@ -56,7 +55,6 @@ import noUnusedSvelteIgnore from '../rules/no-unused-svelte-ignore.js';
 import noUselessChildrenSnippet from '../rules/no-useless-children-snippet.js';
 import noUselessMustaches from '../rules/no-useless-mustaches.js';
 import preferClassDirective from '../rules/prefer-class-directive.js';
-import preferConst from '../rules/prefer-const.js';
 import preferDestructuredStoreProps from '../rules/prefer-destructured-store-props.js';
 import preferStyleDirective from '../rules/prefer-style-directive.js';
 import preferSvelteReactivity from '../rules/prefer-svelte-reactivity.js';
@@ -110,7 +108,6 @@ export const rules = [
     noIgnoredUnsubscribe,
     noImmutableReactiveStatements,
     noInlineStyles,
-    noInnerDeclarations,
     noInspect,
     noNavigationWithoutBase,
     noNavigationWithoutResolve,
@@ -136,7 +133,6 @@ export const rules = [
     noUselessChildrenSnippet,
     noUselessMustaches,
     preferClassDirective,
-    preferConst,
     preferDestructuredStoreProps,
     preferStyleDirective,
     preferSvelteReactivity,
diff --git a/package.json b/package.json
index 9fe4e180aac22bf0275f09d39448b39031a98483..adb0b5f7420082cfedf9306f3b43b46c1ebdb796 100644
--- a/package.json
+++ b/package.json
@@ -40,6 +40,7 @@
     "@jridgewell/sourcemap-codec": "^1.5.0",
     "esutils": "^2.0.3",
     "globals": "^16.0.0",
+	"eslint": "^9.0.0",
     "known-css-properties": "^0.37.0",
     "postcss": "^8.4.49",
     "postcss-load-config": "^3.1.4",
